From d39d5736c985cd65b57bd80acce3c2c827812b57 Mon Sep 17 00:00:00 2001
From: root <root@office.localdomain>
Date: Thu, 12 Mar 2026 16:35:46 -0400
Subject: [PATCH] Fix CheckLinkerFlag source path for submodule builds

---
 cmake/CheckLinkerFlag.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/CheckLinkerFlag.cmake b/cmake/CheckLinkerFlag.cmake
index 7ecf5f6..62e8117 100644
--- a/cmake/CheckLinkerFlag.cmake
+++ b/cmake/CheckLinkerFlag.cmake
@@ -6,7 +6,7 @@ macro(CHECK_LINKER_FLAG flag VARIABLE)
       message(STATUS "Looking for ${flag} linker flag")
     endif()

-    set(_cle_source ${CMAKE_SOURCE_DIR}/cmake/CheckLinkerFlag.c)
+    set(_cle_source "${CHECK_LINKER_FLAG_CMAKE_DIR}/CheckLinkerFlag.c")

     set(saved_CMAKE_C_FLAGS ${CMAKE_C_FLAGS})
     set(CMAKE_C_FLAGS "${flag}")
--
2.34.1

